tools/webpack.assets.json

```diff
@@ -31,5 +31,6 @@
               "./node_modules/plotly.js/dist/plotly-basic.min.js"
              ],
     "translations": "./static/js/translations.js",
-    "zxcvbn": "./node_modules/zxcvbn/dist/zxcvbn.js"
+    "zxcvbn": "./node_modules/zxcvbn/dist/zxcvbn.js",
+    "portico-styles": "./static/styles/scss/portico.scss"
```

Comment:
Is there a useful way to put the CSS part of this into its own file with a separate namespace?  Or do we need to carefully name all the CSS files with `-styles` is in the name to avoid having the JS and CSS bundles with a given name have the same name?  

(Or is the normal thing to do to make the `portico` bundle have both JS and CSS in it?)

Reply:
The ideal scenario is that we import scss files from the associated JS files for the page ( which ideally there would be one per page ). If there are multiple JS files being included in the page we can create a container file that imports those other files and the CSS files. For this PR I opted to add a portico-styles bundle separately because I didn't want to touch JS files associated with all the various places the portico stylesheet was being imported. I think when we decide to go forward we should take the import CSS from JS approach and move this portico file to an associated JS file as well.